<?xml version="1.0"?>
<launch>

    <arg name="projectors" default=""/>
    <arg name="scene_server"/>

    <group ns="/art/interface/projected_gui">

        <include file="$(find art_projected_gui)/launch/common_params.launch">
          <arg name="scene_server" value="$(arg scene_server)"/>
        </include>
    
        <node pkg="art_projected_gui" name="app" type="app.py" output="screen">

            <rosparam param="cursors">[]</rosparam>
            
            <!-- n1 temporarily removed as projector is out of order -->
            <param name="projectors" value="$(arg projectors)"/>

            <!-- open window with scene - for debugging purposes (can be controlled using mouse) -->
            <param name="show_scene" value="false"/>
            <param name="show_touch_points" value="false"/>

        </node>
    
    </group>
    
</launch>
